test/e2e/runner.go

```diff
@@ -56,20 +57,73 @@ func main() {
 	if err != nil {
 		log.Fatalf("Error while trying to build CLI Provisioner:%s", err)
 	}
-	// Only provision a cluster if there isnt a name present
-	if cfg.Name == "" {
-		if cfg.SoakClusterName != "" {
-			rg := cfg.SoakClusterName
-			log.Printf("Deleting Group:%s\n", rg)
+
+	sa := new(azure.StorageAccount)
+	sa.Name = "acsesoaktests" + cfg.Location
+	sa.ResourceGroup.Name = "acse-test-infrastructure-storage"
+	sa.ResourceGroup.Location = cfg.Location
+	err = sa.SetConnectionString()
+	if err != nil {
+		log.Printf("Error while trying to set storage account connection string: %s\n", err)
+	}
+
+	// TODO: create storage account if it doesn't exist
```

Comment:
Is this still a TODO item?

Reply:
Not anymore. I removed it what I meant really was create a file share which is now done. I think it's safe to assume that the account itself pre-exists.